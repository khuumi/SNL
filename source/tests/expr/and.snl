345 and 428.29