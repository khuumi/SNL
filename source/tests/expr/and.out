Binop (Int 345) And (Float 428.29)